### ISSUE TITLE
Multiple clients may not share the same address instance

### ISSUE THREAD

**Issue by lingxiaoyang** [savoirfairelinux/sous-chef#704]
### Expected Behaviour

In sample_data.json, find Paul Taylor and Kevin Tracy:
https://github.com/savoirfairelinux/sous-chef/blob/dev/src/delivery/fixtures/sample_data.json#L24-L34
https://github.com/savoirfairelinux/sous-chef/blob/dev/src/delivery/fixtures/sample_data.json#L81-L91

You'll see that they have the same address instance ID 511. When this happens, there should be some sort of indication on the UI.

### Actual Behaviour

From the Sous-Chef UI, we can't know if Paul and Kevin share the same address instance. And when we modify Paul's address, Kevin's address will also be changed - again, no messages for this behavior. This would cause undesirable results.

What should we expect if this happens? Or we should use OneToOneField to ensure that clients don't share address instances? Please comment.

### Steps to reproduce

* step 1: backup database, and wipe it: `python manage.py flush`
* step 2: load sample data `python manage.py createsuperuser; python manage.py loaddata sample_data`
* step 3: change Paul's address. And observe Kevin's address.

**Comment by lamontfr** [comment]
@lingxiaoyang 
1. I created the sample data and I reused the address instance to save time; this was a mistake I made. I will add an issue to correct the sample data.
2. The ClientWizard (member/views.py L177) creates a new address instance for new clients, so I assume that in the production database each client has his own address instance.
3. The current relationship between the models is : 0..1 Address <-> 0..N Member. This is the case since May 2016. The 0..1 is because a referent (i.e. doctos) is a member that has no address.
4. I think the relationship should be changed to 0..1 Address <-> 1..1 Member.
